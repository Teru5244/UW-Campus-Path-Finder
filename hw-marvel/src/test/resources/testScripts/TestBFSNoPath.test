# Test if BFS works well between two nodes that have no path

LoadGraph np noPath.csv

FindPath np n1 n2