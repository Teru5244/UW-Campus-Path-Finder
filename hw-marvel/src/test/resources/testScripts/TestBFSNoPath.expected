# Test if BFS works well between two nodes that have no path

loaded graph np

path from n1 to n2:
no path found